composer.json

```diff
@@ -32,15 +32,16 @@
     "homepage" : "http://sabre.io/vobject/",
     "license" : "BSD-3-Clause",
     "require" : {
-        "php"          : "^7.1 || ^8.0",
+        "php"          : "^7.4 || ^8.0",
         "ext-mbstring" : "*",
-        "sabre/xml"    : "^2.1"
+        "ext-json"     : "*",
```

Comment:
ToDo: check why this needs to be added.

Reply:
The JSON parser makes uses of `json_decode`. The JSON extension is pretty much bundled everywhere anyway, so it's more about making things sure (and my IDE happy).